internal/venice-common/src/main/resources/avro/PartitionState/v14/PartitionState.avsc

```diff
@@ -186,6 +186,15 @@
         "avro.java.string": "String"
       },
       "default": {}
+    },
```

Comment:
Can you add [compat] to the PR title so we know there is a protocol changes, even though it is Server/DVC only.